bin/buildBlueprints.js

```diff
@@ -137,14 +135,26 @@ build(config, function(stats) {
       '\n   ******************************'
     ));
 
-    var m = new Mocha();
+    m = new Mocha({ reporter: mochaNotifier.decorate('spec') });
     stats.assets.forEach(function(asset) {
-      var path = './.test/' + asset.name;
-      m.addFile(path);
+      m.addFile('./.test/' + asset.name);
+    });
+    m.run();
+
+    // we want to remove these from the require cache while we have path
+    // references to them to ensure they get tested on the next rebuild
+    m.files.forEach(function(filePath) {
+      delete require.cache[require.resolve(path.resolve(filePath))];
     });
-    m.run()
-      .on('end', function() {
-        rimraf('./.test/', function() {});
-      });
+  }
+});
+
+// Hacky way to handle webpacks file output
+process.on('SIGINT', function() {
+  if (argv.runTest) {
+    var testDirectory = configs.DefaultTestingConfig.webpack.output.path;
```

Comment:
True, that would be cleaner, it would require some re-arrangement. Personally this seems okay to me for now, especially as it sounds like we'll come back to this.